VMBackup/main/Utils/HostSnapshotObjects.py

```diff
@@ -1,25 +1,27 @@
 import json
 
 class HostDoSnapshotRequestBody:
-    def __init__(self, taskId, diskIds, settings, snapshotTaskToken, snapshotMetadata):
+    def __init__(self, taskId, diskIds, settings, snapshotTaskToken, snapshotMetadata, instantAccessDurationMinutes = None):
         self.taskId = taskId
         self.diskIds = diskIds
         self.snapshotMetadata = snapshotMetadata
         self.snapshotTaskToken = snapshotTaskToken
         self.settings = settings
+        self.instantAccessDurationMinutes = instantAccessDurationMinutes
 
     def convertToDictionary(self):
-        return dict(taskId = self.taskId, diskIds = self.diskIds, settings = self.settings, snapshotTaskToken = self.snapshotTaskToken, snapshotMetadata = self.snapshotMetadata)
+        return dict(taskId = self.taskId, diskIds = self.diskIds, settings = self.settings, snapshotTaskToken = self.snapshotTaskToken, snapshotMetadata = self.snapshotMetadata, instantAccessDurationMinutes = self.instantAccessDurationMinutes)
 
 class HostPreSnapshotRequestBody:
-    def __init__(self, taskId, snapshotTaskToken, preSnapshotSettings = None):
+    def __init__(self, taskId, snapshotTaskToken, preSnapshotSettings = None, instantAccessDurationMinutes = None):
         self.taskId = taskId
         self.snapshotTaskToken = snapshotTaskToken
         if (preSnapshotSettings != None):
             self.preSnapshotSettings = preSnapshotSettings
+        self.instantAccessDurationMinutes = instantAccessDurationMinutes
```

Comment:
commenting to ensure we have parity with windows changes(to decide if we are going to have this flag/property passed as part of presnapshot request) post testing